Kitodo-API/pom.xml

```diff
@@ -28,4 +28,12 @@
     <artifactId>kitodo-api</artifactId>
     <name>Kitodo - API</name>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

this two classes has automatically imported this dependancy